src/main/java/emu/grasscutter/database/DatabaseHelper.java

```diff
@@ -90,6 +90,10 @@ public static boolean deleteAccount(String username) {
 		return DatabaseManager.getDatastore().find(Account.class).filter(Filters.eq("username", username)).delete().getDeletedCount() > 0;
 	}
 
+	public static List<GenshinPlayer> getAllPlayers() {
+		return DatabaseManager.getDatastore().find(GenshinPlayer.class).stream().toList();
```

Comment:
Thanks for this line of code, it helped me a lot, I was converting wetabq's kt to java, and there was a `toList`, and I found here that I could use `stream`